Source/WebUI/Static/js/lang.js

```diff
@@ -0,0 +1,81 @@
+// lang.js
+// By GoBlock2021
+
+
+// 定义默认语言
+const defaultLang = 'en-US';
+
+
+function saveLanguage(lang) {
```

Comment:
Could you try to match the codestyle that already exists in main.js please to keep things consistent.